### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Sviatoslav3413**
- [DEMO LINK](https://Sviatoslav3413.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Sviatoslav3413.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 1]
```diff
@@ -5,10 +5,59 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <div class="logo">
+        <a href="#home">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
```
Comment (liliia-livandovskaya) [inline src/index.html:21]:
```suggestion
      </div>
      
      <nav class="nav">
```

File: src/index.html [hunk 2]
```diff
@@ -5,10 +5,59 @@
...
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
```
Comment (liliia-livandovskaya) [inline src/index.html:26]:
```suggestion
          </li>
          
          <li class="nav__item">
```
pay attention to the code style, fix it all over the page

File: src/style.css [hunk 3]
```diff
@@ -1 +1,88 @@
...
+  justify-content: end;
+  margin-right: 50px;
+  margin-left: auto;
+}
+
+.nav__list {
+  display: flex;
+  /* justify-content: center; */
+  /* background-color: green; */
+  /* margin-right: 40px; */
```
Comment (liliia-livandovskaya) [inline src/style.css:32]: remove comments

File: src/style.css [hunk 4]
```diff
@@ -1 +1,88 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  text-transform: uppercase;
+  /* background-color: red; */
+  /* margin: 22.5px 10px; */
```
Comment (liliia-livandovskaya) [inline src/style.css:45]: remove comments

File: src/style.css [hunk 5]
```diff
@@ -1 +1,88 @@
...
+  border-style: solid;
+  display: flex;
+  margin-top: 33px;
+  width: 100%;
+  position: absolute;
+}
+
+/* [data-qa="hover"]:hover {
+  color: #00acdc;
+} */
```
Comment (liliia-livandovskaya) [inline src/style.css:88]:
```suggestion

```

File: src/index.html [hunk 6]
```diff
@@ -5,10 +5,59 @@
...
+        <!-- <div class="button">
+          <a class="is-active">APPLE</a>
+        </div>
+        <div class="button">SAMSUNG </div>
+        <div class="button">SMARTPHONES</div>
+        <div class="button" data-qa="hover">LAPTOPS & COMPUTERS</div>
+        <div class="button">GADGETS</div>
+        <div class="button">TABLETS</div>
+        <div class="button">PHOTO</div>
+        <div class="button video">VIDEO</div> -->
```
Comment (liliia-livandovskaya) [inline src/index.html:59]:
```suggestion
```
remove comments

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
GJ! Pay attention to comments

File: src/style.css [hunk 7]
```diff
@@ -1 +1,79 @@
...
+.nav__item:last-child a {
+  margin-right: 0;
+}
+
+a.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+a.is-active::after {
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:68]: why do you need tag selector? use only class selector

File: src/style.css [hunk 8]
```diff
@@ -1 +1,79 @@
...
+  content: "";
+  background-color: #00acdc;
+  height: 0;
+  border-width: 2px;
+  border-radius: 4px;
+  border-style: solid;
+  display: flex;
+  margin-top: 33px;
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:76]: mb just `bottom: 0` suits better?
if we change height of the element we will need to go here and change margin top too(

File: src/style.css (hunk 7 above)
Comment (Bohdan-Maksymiuk) [inline src/style.css:68]: use only class selectors

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
check design  - header have box-shadow

File: src/index.html [hunk 9]
```diff
@@ -5,10 +5,59 @@
...
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones"class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops & computers"class="nav__link"
+              data-qa="hover">Laptops & computers
+            </a>
```
Comment (polosanya) [inline src/index.html:41]:
```suggestion
            <a 
              href="#laptops & computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & computers
            </a>
```

File: src/index.html [hunk 10]
```diff
@@ -5,10 +5,59 @@
...
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung"class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones"class="nav__link">Smartphones</a>
+          </li>
```
Comment (polosanya) [inline src/index.html:36]:
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>

          <li class="nav__item">
            <a href="#smartphones" class="nav__link">Smartphones</a>
          </li>
```
Why no spaces between attributes?
Please fix all cases

File: src/style.css [hunk 11]
```diff
@@ -1 +1,81 @@
...
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-left: 50px;
+  padding-top: 10px;
+  padding-bottom: 10px;
```
Comment (polosanya) [inline src/style.css:20]: You can use shorthand properties instead
https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax

File: src/style.css [hunk 12]
```diff
@@ -1 +1,81 @@
...
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-family: "Roboto", sans-serif;
```
Comment (polosanya) [inline src/style.css:38]: Set font just for the whole page and inherit if needed

File: src/style.css [hunk 13]
```diff
@@ -1 +1,81 @@
...
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  color: black;
+  padding: 22.5px 0;
+  margin: 0 10px;
+}
+
+a:hover {
```
Comment (polosanya) [inline src/style.css:53]: Don't use tags as selectors. Use classes

File: src/style.css [hunk 14]
```diff
@@ -1 +1,81 @@
...
+  content: "";
+  background-color: #00acdc;
+  height: 0;
+  border-width: 2px;
+  border-radius: 4px;
+  border-style: solid;
+  display: flex;
+  bottom: 0;
+  width: 100%;
+  position: absolute;
```
Comment (polosanya) [inline src/style.css:80]: Pay attention to your css properties order
https://9elements.com/css-rule-order/

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost done! Please pay attention to my comments

**Comment by Sviatoslav3413** [comment]
> * [DEMO LINK](https://Sviatoslav3413.github.io/layout_moyo-header/)
> * [TEST REPORT LINK](https://Sviatoslav3413.github.io/layout_moyo-header/report/html_report/)

**Comment by Sviatoslav3413** [comment]
> * [DEMO LINK](https://Sviatoslav3413.github.io/layout_moyo-header/)
> * [TEST REPORT LINK](https://Sviatoslav3413.github.io/layout_moyo-header/report/html_report/)

**Review by liliia-livandovskaya — COMMENTED** [review]
Why have you closed your PR?

**Pull request closed without merging**